neonvm-daemon/pkg/cpuscaling/sys_fs_state_scaling.go

```diff
@@ -0,0 +1,196 @@
+package cpuscaling
```

Comment:
nits:

* Underscores in filenames have semantic meaning in Go, should we avoid that here?
* The current file structure has a centralized `pkg/` directory but individual `component/cmd/` directories. Can we move this to the top-level `pkg/` directory?

Reply:
Moved, renamed